Microsoft.Research/Contracts/MsCorlib/System.Math.cs

```diff
@@ -93,29 +93,40 @@ public static Decimal Abs(Decimal value)
     [Pure]
     public static float Abs(float value)
     {
+      Contract.Ensures((Contract.Result<float>() >= 0.0) || float.IsNaN(Contract.Result<float>()) || float.IsPositiveInfinity(Contract.Result<float>()));
+
+      // 2015-03-26: tom-englert
```

Comment:
:bulb: If the `Contract.Ensures` lines below can be preserved in addition to the newly-inserted `Contract.Ensures`, then this comment can be removed. The new `Ensures` line is correct and there was no loss of information to include it.